massa-protocol-worker/src/handlers/peer_handler/mod.rs

```diff
@@ -304,14 +321,7 @@ impl InitConnectionHandler for MassaHandshake {
                     Some("Failed to deserialize PeerId".to_string()),
                 )
             })?)?;
-        {
-            let peer_db_read = self.peer_db.read();
-            if let Some(info) = peer_db_read.peers.get(&peer_id) {
-                if info.state == PeerState::Banned {
-                    debug!("Banned peer tried to connect: {:?}", peer_id);
-                }
-            }
-        }
+        offset += 32;
```

Comment:
use constants

Reply:
as a followup: remove all the dangerous (panic-prone) deserialization logic

Reply:
It will be refactored to use nom as in the rest of the code in v2